content/docs/CLI/init.md

```diff
@@ -9,7 +9,10 @@ weight: 3
 
 Initiate a repository to start using packit. By default this command adds
 `.packit.yaml` config file to the git repository in the current working
-directory.
+directory.  
+If spec file is present somewhere in your git repository, `init` will set
+the [specfile_path](/docs/configuration/#specfile_path) to it in `.packit.yaml`.
+Otherwise, `init` will set the `specfile_path` to \<the name of the repository\>.spec.
```

Comment:
```suggestion
If a spec file is found in the git repository, `init` will set
[specfile_path](/docs/configuration/#specfile_path) to point to it in `.packit.yaml`.
Otherwise, `specfile_path` is set to `<the name of the repository>.spec`.
```